### PR TITLE
fix: handle `undefined` values in custom properties validator

### PR DESCRIPTION
**Pull request by barmac**
Related to https://camunda.slack.com/archives/CKGH9LR40/p1684417089472889

### CODE REVIEW

**Comment by marcellobarile** [comment]
hello team, any update on this issue? Please note that it's blocking a dependency update on the web-modeler, which is blocking a PR for a bug fix on the webhook tab (in the details panel) :) thanks!

File: src/provider/cloud-element-templates/properties/CustomProperties.js [hunk 1]
```diff
@@ -445,11 +445,11 @@ function propertyValidator(translate, property) {
       return translate('Must not be empty.');
     }
 
-    if (maxLength && value.length > maxLength) {
+    if (maxLength && (value || '').length > maxLength) {
```
[the remark this answers is not in the document]
Comment (barmac) [reply src/provider/cloud-element-templates/properties/CustomProperties.js:448]: I don't think so, at least as long as conditional templates aren't a thing in C7. I could not reproduce the issue without conditional properties.

**Review by nikku — APPROVED** [review]

**Comment by barmac** [comment]
This will be released as v1.24.1 pretty soon.

**Comment by marcellobarile** [comment]
thanks for the prompt response!! :)

**Pull request merged**